server/checks/Approval.js

```diff
@@ -467,6 +467,11 @@ export default class Approval extends Check {
           debug(`${repository.full_name}#${issue.number}: Ignoring comment, not a PR`)
           return
         }
+        const author = hookPayload.comment.user.login
+        if (author.endsWith('-robot')) {
```

Comment:
This looks like a hack, shouldn't this be configurable?

Reply:
Yeah, or maybe a list of suffixes or regexes would be great.